solidity/random-beacon/contracts/libraries/Relay.sol

```diff
@@ -192,6 +192,29 @@ library Relay {
         return (inactiveMembers, slashingAmount);
     }
 
+    /// @notice Validates if the reporting of an unauthorized group signing can
+    ///         be performed. The signature verification is done by BLS lib.
+    /// @param signedMsgSender Signature of the sender's address as a message.
+    /// @param group Group that is being verified against private key leakage.
+    /* solhint-disable no-unused-vars */
```

Comment:
Why do we need it? I think we don't need to pass `self`.

Reply:
it's gone now